### PR TITLE
PoET propagation when not synced

### PR DESCRIPTION
**Pull request by noamnelke**

### CODE REVIEW

File: activation/poetlistener.go [hunk 1]
```diff
@@ -8,10 +8,15 @@ import (
 
 const PoetProofProtocol = "PoetProof"
 
+type PoetDbI interface {
```
Comment (y0sher) [inline activation/poetlistener.go]: didn't like the name 👎

**Review by y0sher — APPROVED** [review]
Did we decide to not test that we're still sending poetproof when not synced ?

**Comment by noamnelke** [comment]
@y0sher I meant to make this PR a draft, I want to add an assertion to the late nodes test before merging this

File: activation/poetlistener.go (hunk 1 above)
Comment (noamnelke) [reply activation/poetlistener.go]: Yeah, it's 💩 I'll find something better

**Pull request merged**